nailgun/entities.py

```diff
@@ -2457,6 +2457,8 @@ class OperatingSystem(
     * architecture
```

Comment:
1) ok, will take a look and update
2) That bug 5 lines above is more like exclusion - take a brief look at whole file, in most cases we include full link. And also in the end it becomes beautiful short link like this:
![2015-12-10 13_41_42-fedora 23 xfce light - vmware workstation](https://cloud.githubusercontent.com/assets/11719030/11715851/16ee8eec-9f4e-11e5-8e2c-ee53505acfeb.png)
3) any ideas how?